cmd/staticlint/internal/osexitchecker/osexitchecker.go

```diff
@@ -6,6 +6,8 @@ import (
 	"golang.org/x/tools/go/analysis"
 )
 
+var mainName = "main"
```

Comment:
Если это из-за линтера, то лучше

```suggestion
const mainName = "main"
```